libs.versions.toml

```diff
@@ -0,0 +1,161 @@
+# Also see settings.gradle file for dynamic versionCatalogs dependencies
+
+[versions]
+antlrVersion = "4.8"
+awsSdkVersion = "2.20.18"
+flink115Version = "1.15.0"
+flink116Version = "1.16.2"
+flink117Version = "1.17.1"
+hadoop2Version = "2.7.3"
+hive2Version = "2.3.9"
+hive3Version = "3.1.3"
+jackson211Version = "2.11.4"
+jackson212Version = "2.12.3"
+jackson213Version = "2.13.4"
+jacksonVersion = "2.14.2"
+jettyVersion = "9.4.43.v20210629"
+junitJupiterVersion = "5.9.2"
+mockitoVersion = "4.11.0"
+nessieVersion = "0.60.1"
+orcVersion = "1.8.3"
+parquetVersion = "1.13.1"
+springBootVersion = "2.5.4"
+tez010Version = "0.10.2"
+tez08Version = "0.8.4"
+
+[libraries]
+activation = "javax.activation:activation:1.1.1"
+aircompressor = "io.airlift:aircompressor:0.24"
+aliyun-sdk-oss = "com.aliyun.oss:aliyun-sdk-oss:3.10.2"
+antlr-antlr4 = { module = "org.antlr:antlr4", version.ref = "antlrVersion" }
+antlr-runtime = { module = "org.antlr:antlr4-runtime", version.ref = "antlrVersion" }
+arrow-memory-netty = "org.apache.arrow:arrow-memory-netty:11.0.0"
+arrow-vector = "org.apache.arrow:arrow-vector:11.0.0"
+avro-avro = "org.apache.avro:avro:1.11.1"
+awssdk-apache-client = { module = "software.amazon.awssdk:apache-client", version.ref = "awsSdkVersion" }
```

Comment:
Hint: it's easier to use the 
```
awssdk-bom = { module = "software.amazon.awssdk:bom", version = "2.20.18" }
```
and reference the BOM in the build scripts and omit the versions for the individual artifacts. For example
```kotlin
    implementation(platform(libs.awssdk.bom))
    implementation("software.amazon.awssdk:apache-client")
```

Reply:
done